dev

```diff
@@ -0,0 +1 @@
+python3 -m venv venv; source venv/bin/activate; python setup.py develop; export DJANGO_SETTINGS_MODULE=tests.settings;
```

Comment:
I think its better just to put this in the docs as opposed to having a mysterious file people source